utils/plex.py

```diff
@@ -74,8 +75,41 @@ def remove_user(plex_account, user_email):
         logging.info(f"PLEXAPI: Removed {user_email} from shares on account: {plex_account.email}")
         return True
     except NotFound:
-        logging.error(f"PLEXAPI: Cannot remove {user_email} from shares on account: {plex_account.email}: not sharing with this user")
+        logging.error(
+            f"PLEXAPI: Cannot remove {user_email} from shares on account: {plex_account.email}: not sharing with this user")
```

Comment:
I am once again asking you to stop wrapping my code

Reply:
no